grade.go

```diff
@@ -392,39 +404,27 @@ func main() {
 
 		result.StudentId = hydratedStudent.Id
 		result.AssignmentId = int8(1)
-
-		if language == "python3" || language == "python" {
+		
+		if language == "python3" || language == "python" || language == "javascript" || language == "js" {
```

Comment:
may be worth writing a [`contains(ss []string, t string) bool`](https://stackoverflow.com/a/10485970/7543054) method

Reply:
Excellent idea, that would likely make this if/else more maintainable.